x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -106,63 +116,70 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
     }),
   },
   enrich: {
-    FieldsComponent: undefined, // TODO: Implement
+    FieldsComponent: Enrich,
     docLinkPath: '/enrich-processor.html',
```

Comment:
Forgot to add this in my original review:

Unlike the other processors, the enrich processor is a bit complicated. I'd link to these docs rather than the processor docs:
```suggestion
    docLinkPath: '/ingest-enriching-data.htmll',
```